homeassistant/components/braviatv/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "braviatv",
   "name": "Sony Bravia TV",
   "documentation": "https://www.home-assistant.io/integrations/braviatv",
-  "requirements": ["pybravia==0.1.3"],
+  "requirements": ["pybravia==0.1.5"],
```

Comment:
It looks like the library is defaulting to not raising exceptions.

Reply:
By default, no. I thought that the result of executing the connect method (returns bool) is enough to understand whether it was connected or not. If necessary, I can throw an exception by default, but do I need to handle it somehow on the HA side?

Reply:
@MartinHjelmare Now the library raising exceptions on connect by default. Following [this guide](https://developers.home-assistant.io/docs/integration_fetching_data/#coordinated-single-api-poll-for-data-for-all-entities) I raise `UpdateFailed` in `_async_update_data` on error. Please take a look.